python/mlc_llm/serve/engine_base.py

```diff
@@ -199,7 +199,7 @@ def _estimate_mem_usage_and_max_total_sequence_length(  # pylint: disable=too-ma
     if gpu_size_bytes is None:
         raise ValueError("Cannot read total GPU global memory from device.")
     if gpu_memory_utilization is None:
-        gpu_memory_utilization = 0.90
```

Comment:
Do we have increased memory usage?

Reply:
GPUSampler allocates GPU memory for the draft prob distributions which increases the risk of OOM. Reducing the default values is a safe choice here, and meanwhile we can also enhance the estimation.